### PR TITLE
Used the random_name function in the Default implementation of NodeManagerDefaults

### LINKED ISSUE #6382: Use the `random_name` function in the `Default` implementation of `NodeManagerDefaults`
## Current behavior

The `node_name` field of the `Default` implementation of the `NodeManagerDefaults` struct is calling `hex::encode()` to generate a random name.

## Desired behavior

The `node_name` field of the `Default` implementation of the `NodeManagerDefaults` struct should be calling the [`random_name()`](https://github.com/build-trust/ockam/blob/75fa2e3ece6c8ece4a89056e5360d3b1ca19ddeb/implementations/rust/ockam/ockam_api/src/cli_state/mod.rs#L421) function.

## Implementation details

This is the field that must be changed:

https://github.com/build-trust/ockam/blob/140a8bd71183754d70ec1e4bc1a5c962062fa0a9/implementations/rust/ockam/ockam_api/src/nodes/service/in_memory_node.rs#L178

---

<i>

Some tips for newcomers:
1. Create a new branch for the issue you are working on (don't push directly to your fork's `develop` branch)
2. Run `cargo fmt`. You can configure the editor to run it automatically everytime a file is saved
3. Run `cargo clippy` to make sure the code compiles + follows rust standards/best practices
4. You don't need to keep your branch updated with `develop` until there is a merge conflict. If you still want to update it, use `git rebase develop` instead of `gir merge develop` 

We love helping new contributors! ❤️
If you have questions or need help as you explore, please join us on [Discord](https://discord.gg/bewvnm6zqS). If you're looking for other issues to contribute to, please checkout our [good first issues](https://github.com/build-trust/ockam/labels/good%20first%20issue).

</i>

### PR DESCRIPTION
**Pull request by Virtual4087**
Closes #6382 
## Checks

- [x] All commits in this Pull Request are [signed](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) and Verified by Github.
- [x] All commits in this Pull Request follow the Ockam [commit message convention](https://github.com/build-trust/.github/blob/main/CONTRIBUTING.md#commit-messages).
- [x] There are no Merge commits in this Pull Request. Ockam repo maintains a linear commit history. We merge Pull Requests by rebasing them onto the develop branch. Rebasing to the latest develop branch and force pushing to your Pull Request branch is okay.
- [x] I have read and accept the Ockam Community [Code of Conduct](https://github.com/build-trust/.github/blob/main/CODE_OF_CONDUCT.md).
- [x] I have read and accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/.github/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam/blob/develop/.github/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam](https://github.com/build-trust/ockam) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam/edit/develop/.github/CONTRIBUTORS.csv) file in the github web UI and create a separate Pull Request, this will mark the commit as verified.

### CODE REVIEW

File: implementations/rust/ockam/ockam_api/src/nodes/service/in_memory_node.rs [hunk 1]
```diff
@@ -18,6 +18,7 @@ use crate::nodes::{NodeManager, NODEMANAGER_ADDR};
 use crate::session::sessions::{Key, Session};
 use crate::session::MedicHandle;
 use crate::DefaultAddress;
+use crate::cli_state::random_name;
```
Comment (adrianbenavides) [inline implementations/rust/ockam/ockam_api/src/nodes/service/in_memory_node.rs]: Make sure to run cargo clippy, there is an unused import you need to remove 👍

**Comment by adrianbenavides** [comment]
Hey @Virtual4087 , the PR looks good to merge but we need you to accept our CLA as described here:

> I have read and accepted the Ockam [Contributor License Agreement](https://github.com/build-trust/.github/blob/main/CLA.md) by adding my Git/Github details in a row at the end of the [CONTRIBUTORS.csv](https://github.com/build-trust/ockam/blob/develop/.github/CONTRIBUTORS.csv) file in a separate pull request to the [build-trust/ockam](https://github.com/build-trust/ockam) repository. The easiest way to do this is to [edit the CONTRIBUTORS.csv](https://github.com/build-trust/ockam/edit/develop/.github/CONTRIBUTORS.csv) file in the github web UI and create a separate Pull Request, this will mark the commit as verified.

Once the CLA PR is merged, we'll be able to merge this one 👍

**Comment by Virtual4087** [comment]
So i need to create a new branch, add my info in CONTRIBUTORS.csv file and then create a new pr, right?

**Comment by adrianbenavides** [comment]
> So i need to create a new branch, add my info in CONTRIBUTORS.csv file and then create a new pr, right?

That's right! I see you already created it, it will get merged soon, thanks.

**Review by adrianbenavides — APPROVED** [review]
Looks like everything is ready now. Thank you so much @Virtual4087 for your PR, will get merged soon! 🚂

**Review by etorreborre — APPROVED** [review]

**Pull request merged**